esx_service/vmdk_ops_test.py

```diff
@@ -380,7 +388,10 @@ class VmdkAttachDetachTestCase(unittest.TestCase):
 
     volNamePre = "vol_UnitTest_Attach"
     vm_name = 'test-vm'
-    max_vol_count = 60
+    if config["run_max_attach"]:
+        max_vol_count = MAX_VOL_COUNT_FOR_ATTACH
```

Comment:
Good point, we should use uniittest features to turn on/off tests. How about I drop this part from here and submit a separate PR with test selection?